app/views/shared/_search_result.html.erb

```diff
@@ -6,67 +6,73 @@
         <span><%= t(".labels.status") %></span>
         <span class="<%= "status-tag-#{status_tag_for(result)}" %>"><%= t(".statuses.#{status_tag_for(result)}") %></span>
       </li>
+
       <% if result.operator_name.present? %>
-      <li>
-        <span><%= t(".labels.operator_name") %></span>
-        <%= result.operator_name %>
-      </li>
+        <li>
+          <span><%= t(".labels.operator_name") %></span>
+          <%= result.operator_name %>
+        </li>
       <% end %>
+
       <% if result.reference.present? %>
-      <li>
-        <span><%= t(".labels.reference") %></span>
-        <%= result.reference %>
-      </li>
+        <li>
+          <span><%= t(".labels.reference") %></span>
+          <%= result.reference %>
+        </li>
       <% end %>
     </ul>
   </div>
   <div class="column-one-quarter">
     <% if result.site_address.present? %>
       <ul class="registration-details">
         <% if result.site_address.postcode.present? %>
-        <li>
-          <span><%= t(".labels.address") %></span>
-          <ul>
-            <% displayable_address(result.site_address).each do |address_line| %>
-              <li><%= address_line %></li>
-            <% end %>
-          </ul>
-        </li>
+          <li>
+            <span><%= t(".labels.address") %></span>
+            <ul>
+              <% displayable_address(result.site_address).each do |address_line| %>
+                <li><%= address_line %></li>
+              <% end %>
+            </ul>
+          </li>
         <% end %>
+
         <% if result.site_address.grid_reference.present? %>
-        <li>
-          <span><%= t(".labels.grid_reference") %></span>
-          <%= result.site_address.grid_reference %>
-        </li>
+          <li>
+            <span><%= t(".labels.grid_reference") %></span>
+            <%= result.site_address.grid_reference %>
+          </li>
         <% end %>
       </ul>
     <% else %>
-      <br>
+      <br />
```

Comment:
Minor tweak, and though valid its our understanding that void elements should just be defined using the opening tag syntax

```suggestion
      <br>
```

See
- <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br>
- <https://dev.w3.org/html5/html-author/#void>
- <https://stackoverflow.com/a/1946446>